Packs/Microsoft365Defender/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Microsoft 365 Defender",
     "description": "Microsoft Defender XDR (formerly Microsoft 365 Defender) is a unified pre- and post-breach enterprise defense suite that natively coordinates detection, prevention, investigation, and response across endpoints, identities, email, and applications to provide integrated protection against sophisticated attacks.",
     "support": "xsoar",
-    "currentVersion": "4.6.2",
+    "currentVersion": "4.7.0",
```

Comment:
this may considered as minor change 
so the value should be `4.6.3`
and the release notes file should be changed according